app/templates/releases/lts.hbs

```diff
@@ -52,6 +52,6 @@
   </table>
 
   <p>
-    These LTS versions are no longer maintained: 3.4 (Jan 2019), 2.18 (Feb 2018), 2.16 (Feb 2018), 2.12 (Apr 2017), 2.8 (Nov 2016), and 2.4 (Apr 2016). Their last minor release dates are shown in parentheses.
+    These LTS versions are no longer maintained: 3.8 (Aug 2019), 3.4 (Jan 2019), 2.18 (Feb 2018), 2.16 (Feb 2018), 2.12 (Apr 2017), 2.8 (Nov 2016), and 2.4 (Apr 2016). Their last minor release dates are shown in parentheses.
```

Comment:
The last minor version for Ember `v3.8` was `v3.8.3`, which was released on August 5, 2019.

https://github.com/emberjs/ember.js/releases/tag/v3.8.3